types/react-map-gl/index.d.ts

```diff
@@ -309,6 +310,13 @@ export interface NavigationControlProps extends BaseControlProps {
 
 export class NavigationControl extends BaseControl<NavigationControlProps> {}
 
+export interface FullscreenControlProps extends BaseControlProps {
+    className?: string;
+    container?: object | null;
```

Comment:
Is this maybe `container?: HTMLElement`? Not sure why null is accepted here.

Reply:
I followed the propTypes used in the original implementation: https://github.com/uber/react-map-gl/blob/4.1-release/src/components/fullscreen-control.js
Notice that the defaultProp value is `null`

Reply:
Ah, interesting. In that case I think `HTMLElement | null` is still the most correct thing here, especially since prop-types package does not provide html element prop type, but we can get that with ts.

Reply:
Allright, seems reasonable

Reply:
HTMLElement matches with mapbox-gl-js definition also: https://docs.mapbox.com/mapbox-gl-js/api/#fullscreencontrol so I'll update it.